R/class-plotdata-map-markers.R

```diff
@@ -91,26 +89,35 @@ newMapMarkersPD <- function(.dt = data.table::data.table(),
       xRange <- findViewport(.pd[[x]], xType)
       .pd[[x]] <- as.character(bin(.pd[[x]], binWidth, xRange))
       veupathUtils::logWithTime('Successfully binned continuous x-axis.', verbose)
+
+      # maybe worth at some point a fxn getRankedValues(x = character(), maxNumValues = integer(), otherBin = c(T,F))
+      # if maxNumValues was exceeded and otherBin = F then it would just put out an error
+      # maxNumValues could be NULL by default maybe?
+      # would something like that be better here or veupathUtils (could mbio use it?)
```

Comment:
Oh i like it. Not sure mbio has a use for it at the moment, but sounds more like a general use function, especially with defaults maxNumValues=Null and otherBin=F.